doc/source/using_config.rst

```diff
@@ -145,6 +145,9 @@ toplevel of your configuration file, like so:
      # are used.
      connection-config:
        keepalive-time: 60
+       retry-limit: 4
+       retry-delay: 1000
+       request-timeout: 300
```

Comment:
It would be a good idea to document the units given that retry-delay is in milliseconds while the other durations are in seconds.

Reply:
I've improved the documentation.